src/JitInterface/src/CorInfoTypes.cs

```diff
@@ -1159,7 +1159,14 @@ public struct SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR
         public byte eightByteOffsets1;
     };
 
-    // DEBUGGER DATQA
+    // DEBUGGER DATA
+    public enum ICorDebugInfo
```

Comment:
The unmanaged name of the enum is CorDebugIlToNativeMappingTypes - we should name it same or similar.